programs/drift/src/controller/liquidation.rs

```diff
@@ -1888,6 +1910,18 @@ pub fn liquidate_perp_pnl_for_deposit(
         return Err(ErrorCode::InvalidLiquidation);
     }
 
+    if safest_tier_perp_liability < contract_tier
+        || safest_tier_spot_liability < AssetTier::default()
```

Comment:
`safest_tier_spot_liability < AssetTier::default()` is always true? why are you checking that

Reply:
when user has no liabilities in spot its false, but changed/cleaned this to be inside ContractTier impl